src/alto-ui/SideNav/SideNav.js

```diff
@@ -36,6 +36,7 @@ class SideNav extends React.PureComponent {
       <Link
         id={`${this.props.id}__${item.id}__link`}
         href={item.url}
+        onClick={!!this.state.open && this.handleToggleOpen}
```

Comment:
I think 
`onClick={this.handleToggleOpen}`
it's enough no ?

Reply:
then it fires and changes state.open even if browser isn't small, so when you resize it smaller the menu can be set to open. I thought it should be closed by defaut when browser is resizing horizontally and menu goes up. That's why I added a condition